client/src/pages/UserLanding/UserLanding.jsx

```diff
@@ -34,9 +34,42 @@ const UserLanding = () => {
   // if i have time, consider implementing a token as the dependency for when this function gets called
   useEffect(() => {
     addUserToDB();
+    getSavedWorkouts();
     // console.log("api post");
   }, [user]);
 
+  // get saved workouts from DB
+  const getSavedWorkouts = async () => {
```

Comment:
Would  recommend having this be defined before being used in the useEffect above